contracts/src/TeleporterTokenDestination.sol

```diff
@@ -69,9 +109,25 @@ abstract contract TeleporterTokenDestination is
         require(
             feeTokenAddress_ != address(0), "TeleporterTokenDestination: zero fee token address"
         );
+        require(decimalsShift <= 18, "NativeTokenDestination: invalid decimalsShift");
         sourceBlockchainID = sourceBlockchainID_;
         tokenSourceAddress = tokenSourceAddress_;
         feeTokenAddress = feeTokenAddress_;
+        tokenMultiplier = 10 ** decimalsShift;
+        multiplyOnReceive = multiplyOnReceive_;
+    }
+
+    /**
+     * @dev Scales `value` based on `tokenMultiplier` and the direction of the transfer.
+     * Should be used for all tokens being transferred to/from other subnets.
+     */
+    function scaleTokens(uint256 value, bool isReceive) public view returns (uint256) {
```

Comment:
It could be, but similar to other `view`/`pure` helpers, I think it's preferable to leave them as public so that they can be used as utilities (and also in unit tests).

Reply:
if we're using them in tests I'm good with leaving it as `public`. I noticed that in the tests we write a separate logic/function here though, would we be able to just use this function? https://github.com/ava-labs/teleporter-token-bridge/blob/main/contracts/test/NativeTokenDestinationTests.t.sol#L548